### PR TITLE
Adding getting started page to digital.arizona.edu

### PR DESCRIPTION
**Pull request by camikazegreen**
This is a work in progress, but any early thoughts and review would be appreciated!

Looking at Drupal.org's page as a comp: https://www.drupal.org/download

### CODE REVIEW

File: docs/getting-started.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            <p>There is a default <a href="https://github.com/az-digital/az-quickstart-scaffolding">Quickstart Scaffolding</a> that is used by Lando and DDEV for local development. This can be used as a template for setting up your own server environment.
+            <h2 class="text-blue">Managing Quickstart with Composer</h2>
+            <p>One big change from Drupal 7 to Drupal 8 (and 9) is the adoption of <a href="https://getcomposer.org/">Composer</a> for managing dependencies. Contrib modules are now treated similarly to other PHP packages and can now be included through a composer.json file. When Composer runs, it will pull in all of the correct versions of each dependency and install them in your project. </p>
+
+            <p>In order to use Composer, it will need to be installed on the server hosting your website, or on seperate CI infrastructure.</p>
+
+            <p>Drupal.org provides an in depth <a href="https://www.drupal.org/docs/user_guide/en/install-composer.html">guide to using Composer to manage Drupal projects</a>.
```
Comment (trackleft) [inline docs/getting-started.html:59]: in-depth

**Review by trackleft — COMMENTED** [review]
I think we need to signify that this is developer documentation, not intended for anything other than building and deploying development and unsupported independent sites.

**Comment by michaelhagedon** [comment]
As someone just learning about the QS2 environment, I think this already provides useful context even beyond the helpful Contributing document. Even answers some questions I hadn't figured out how to articulate yet, like the role of the scaffolding repository. 👍

**Comment by smayden** [comment]
There is a typo in the word separate:

"In order to use Composer, it will need to be installed on the server hosting your website, or on seperate CI infrastructure."

**Comment by mmunro-ltrr** [comment]
Should the link on the index.html page get updated o the same PR?

**Review by mmunro-ltrr — APPROVED** [review]

**Comment by camikazegreen** [comment]
When this has enough approvals, I think that we should merge it when we are ready for pre-release.

**Review by danahertzberg — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Review by jwmoore1 — APPROVED** [review]

**Review by mmunro-ltrr — APPROVED** [review]

**Pull request merged**